doc/source/authentication.rst

```diff
@@ -208,6 +208,31 @@ and obtain the confidential client credentials.
          userdata_params: {'state': 'state'}
          username_key: preferred_username
 
+Auth0
+^^^^^
+
+Auth0 is a popular commercial provider of identity management. The JupyterHub helm chart does not include support for
+Auth0 by default.  To use Auth0, ``extraEnv`` and ``extraConfig`` must be configured as follows:
+
+Note that without the scope defined, authenticating to JupyterHub after already being logged in to Auth0 will fail.
+
+.. code-block:: yaml
+
+    hub:
+      extraEnv:
+        AUTH0_SUBDOMAIN: 'prod-8ua-1yy9'
```

Comment:
It looks like `extraEnv` needs to be in `name:`, `value:` format.

```suggestion
      - name: AUTH0_SUBDOMAIN
        value: 'prod-8ua-1yy9'
```

Reply:
This Helm chart supports the current format alongside the name/value format with is the k8s native format.